GoogleVision/src/main/java/org/example/ReceiptMaster.java

```diff
@@ -198,16 +203,19 @@ private Map<String, String> getAssociatedCategories(String text) {
             }
             prevLine = tLine;
         }
-        kdv = kdv.replace(".", "");
-        kdv = kdv.substring(0, kdv.length() - 2) + "." + kdv.substring(kdv.length() - 2);
-        toplam = toplam.replace(".", "");
-        toplam = toplam.substring(0, toplam.length() - 2) + "." + toplam.substring(toplam.length() - 2);
-        tutar = String.valueOf(Integer.parseInt(toplam.replace(".", "")) -
-                Integer.parseInt(kdv.replace(".", "")));
-        tutar = tutar.substring(0, tutar.length() - 2) + "." + tutar.substring(tutar.length() - 2);
-
-        tarih = tarih.replace(".", "/");
-        tarih = tarih.replace("-", "/");
+        try {
```

Comment:
for NullpPointerExceptions we can use java.commons util classes for example:
 if (StringUtils.isNotEmty(kdv)) {
kdv.anyMethod();
}